packages/mui-material/src/Autocomplete/Autocomplete.d.ts

```diff
@@ -159,7 +159,7 @@ export interface AutocompleteProps<
   /**
    * Props applied to the Listbox element.
    */
-  ListboxProps?: ReturnType<ReturnType<typeof useAutocomplete>['getListboxProps']>;
+  ListboxProps?: ReturnType<ReturnType<typeof useAutocomplete<any>>['getListboxProps']>;
```

Comment:
Why this change and why it is applied only in `ListboxProps` prop?

Reply:
After committing changes for the `AutocompleteValue` type, I found the type of the `ListboxProps` prop was accidentally updated to `any`(not `object`) by executing `yarn proptypes && yarn docs:api`, so I changed this for consistency.